smoke_test/run_smoke_test.sh.in

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+KOKKOSKERNELS_INTALL="@CMAKE_BINARY_DIR@"
```

Comment:
KOKKOSKERNELS_INTALL -> KOKKOSKERNELS_INSTALL

Reply:
Actually it seems that this variable is not used anymore so I will just delete it.